content/blog/2020/2020-04-15-covid-19-part-3.md

```diff
@@ -10,6 +10,7 @@ categories = ["covid-19", "visualization", "addons", "trends", "time"]
 shortExcerpt = "Inspection of Covid-19 time trends."
 longExcerpt = "Inspecting and comparing Covid-19 time trends, absolute and relative growth."
 x2images = true
+alias = "2020/2020-4-015-covid-19-part-3/"
```

Comment:
Did this work with your version of hugo? I needed to to `aliases = ["/blog/2020/2020-4-015-covid-19-part-3/"]` to make it work.

Reply:
Yes. 😮 I have tested it.